app/assets/stylesheets/report.scss

```diff
@@ -0,0 +1,32 @@
+.report {
+  background: #fff;
+  border-radius: 2px;
+  display: block;
```

Comment:
Properties should be ordered background, border-radius, box-shadow, display, float, margin, min-height, position, transition, width